lib/tag-generator.js

```diff
@@ -10,20 +11,16 @@ export default class TagGenerator {
   }
 
   getPackageRoot() {
-    let packageRoot
-    if (typeof snapshotResult === 'undefined') {
-      packageRoot = path.resolve(__dirname, '..');
+    if (fs.isAbsolute(__dirname)) {
+      return path.resolve(__dirname, '..')
```

Comment:
@as-cii On Atom master, I'm seeing this method return a path inside the asar bundle: `/Applications/Atom.app/Contents/Resources/app.asar/node_modules/symbols-view`, which is causing an error when attempting to spawn the `ctags` executable. I believe `fs.isAbsolute(__dirname)` is returning true.

I don't quite understand how the `packageRoot` logic works, but it seems like we need to fall through to the `else` branch in the case where we're using asar but no snapshot.